src/video_core/renderer_opengl/gl_shader_manager.cpp

```diff
@@ -368,17 +357,16 @@ class ShaderProgramManager::Impl {
     ShaderDiskCache disk_cache;
 };
 
-ShaderProgramManager::ShaderProgramManager(Frontend::EmuWindow& emu_window_, const Driver& driver_,
-                                           bool separable)
+ShaderProgramManager::ShaderProgramManager(Frontend::EmuWindow& emu_window_, const Driver& driver_)
     : emu_window{emu_window_}, driver{driver_},
-      strict_context_required{emu_window.StrictContextRequired()}, impl{std::make_unique<Impl>(
-                                                                       separable)} {}
+      strict_context_required{emu_window.StrictContextRequired()}, impl{std::make_unique<Impl>()} {}
 
 ShaderProgramManager::~ShaderProgramManager() = default;
 
 bool ShaderProgramManager::UseProgrammableVertexShader(const Pica::Regs& regs,
                                                        Pica::Shader::ShaderSetup& setup) {
-    PicaVSConfig config{regs.vs, setup};
+    PicaVSConfig config{regs.rasterizer, regs.vs, setup, !GLES || GLAD_GL_EXT_clip_cull_distance,
```

Comment:
```suggestion
    PicaVSConfig config{regs.rasterizer, regs.vs, setup, !driver.IsOpenGLES() || driver.HasExtClipCullDistance(),
```
Another idea would to be rename HasExtClipCullDistance to HasClipCullDistance and perform this condition in that function as a more general "implementation supports clip planes"